src/cpp/aiebu/src/common/utils.h

```diff
@@ -144,11 +145,47 @@ inline std::string get_pagelabel(const std::string& label)
 // Custom stream buffer that reads from a vector<char>
 class vector_streambuf : public std::streambuf {
 public:
-    vector_streambuf(const std::vector<char>& vec) {
+    explicit vector_streambuf(const std::vector<char>& vec) {
         char* begin = const_cast<char*>(vec.data());
         this->setg(begin, begin, begin + vec.size());
     }
 };
 
+namespace utilities {
+inline std::vector<const char*>
+vector_of_string_to_vector_of_char(const std::vector<std::string>& args)
+{
+  std::vector<const char*> char_vec;
+  for (auto& arg : args)
+    char_vec.push_back(arg.c_str());
+
+  return char_vec;
+}
+}
+
+enum class fragment{
+  BEGIN_ANCHOR_RE,
+  END_ANCHOR_RE,
+  HEX_RE,
+  L_BRACK_RE,
+  R_BRACK_RE,
+};
```

Comment:
enum class enumerators should be lower case
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#enum5-dont-use-all_caps-for-enumerators